RELEASE.md

```diff
@@ -89,6 +89,20 @@ To release a new version of Spyder you need to follow these steps:
 
 * Release a new version of `python-lsp-server`, if required.
 
+* Merge PRs on Conda-forge that update the `spyder-kernels` and `python-lsp-server` feedstocks (usually an automatic PR will appear that can be either merged as it is or be use as boilerplate):
+
+  - `spyder-kernels`: https://github.com/conda-forge/spyder-kernels-feedstock
+
+  - `python-lsp-server`: https://github.com/conda-forge/python-lsp-server-feedstock
+
+  **Notes**:
+
+  - Review carefully the release notes of those packages to see if it's necessary to add new dependencies or update the constraints on the current ones (e.g. `jedi >=0.17.2`).
+  - After merging each of those PRs, give a ping to the Anaconda team telling them that these packages are required for the new Spyder version. You need to use the handle `@anaconda-pkg-build` for that. Here is an example of this kind of messages:
+
+    https://github.com/conda-forge/spyder-kernels-feedstock/pull/58#issuecomment-725664085
```

Comment:
Please remove this bullet entry. It was useful a year ago, but I doubt the Anaconda team is monitoring that Github handle now.